### ISSUE TITLE
Token selection hint

### ISSUE THREAD

**Issue by Costa769** [bancorprotocol/carbon-app#1106]
### Overview 

Currently, when a user selects a token for trading (as well as in strategy creation and explorer), we do not provide any specific hints to explain which address the selected token corresponds to. This becomes an issue when there are multiple tokens sharing the same symbol, even if they have different logos.

Therefore, if there are several tokens sharing the same symbol, we aim to expose the addresses associated with each of those tokens.

### Design
[mockup](https://www.figma.com/file/YpsatcY9jtxJr0SJwlFYNj/Carbon-Screen-Design?type=design&node-id=13955%3A45701&mode=design&t=uUZC3Fxyb8lWHXlu-1)

### Implementation logic

The following logic should be implemented for both single and token pair selection lists on trade, explorer, simulator and strategy creation pages:

**IF** there are multiple tokens with the exact same symbol in the token selection list:
```
example 1, case TRUE:
token1 = abc
token2 = abc
example 2, case FALSE:
token1 = abc
token2 = abcd
```

- Display the first and last 6 digits of the token address beneath each token sharing the same symbol.
- IF multiple token pairs have at least one token with the same symbol, show  both token addresses of each pair [mockup](https://www.figma.com/file/YpsatcY9jtxJr0SJwlFYNj/Carbon-Screen-Design?type=design&node-id=13955%3A45701&mode=design&t=uUZC3Fxyb8lWHXlu-1).
- Upon hovering over the warning icon, display a tooltip with a token's address or, in the case of a token pair, the addresses of both tokens.[mockup](https://www.figma.com/file/YpsatcY9jtxJr0SJwlFYNj/Carbon-Screen-Design?type=design&node-id=13955%3A45794&mode=design&t=uUZC3Fxyb8lWHXlu-1)
-- Upon clicking the link icon next to an address in the tooltip, open a new tab with the token address on etherscan.

**Comment by tiagofilipenunes** [comment]
Blocked by #1385 as the trade page will move from a pair selection to an individual token selection.

**Comment by tiagofilipenunes** [comment]
Duplicate of #1029
